03_how_ln_works.asciidoc

```diff
@@ -680,6 +680,15 @@ This must hold true for every single channel that the payment is routed through.
 The lowest capacity of a channel set the upper limit of the full route through this channel. 
 Hence, capacity and connectivity are crucial resources in the Lightning network. 
 
+==== Incentives for Large Transaction Value vs. Incentives for Small Transaction Values
+
+The fee structure in Bitcoin is independent of the transaction value. 
+A $1 million transaction has the same fee as a $1 transaction on Bitcoin. 
+In Lightning the fee is a percentage of the transaction value. 
+These opposing fee structures create different incentives and lead to different usage in regards to transaction value.
+A transaction of enough value will be cheaper on Bitcoin and hence users will prefer Bitcoin for large value transactions.
+Similarly, on the other end of the spectrum, users will prefer Lightning for small value transactions.
```

Comment:
this ignores the base fee that lightning has. maybe add another sentence?

Reply:
improved according to your suggestion
in commit # 2
cf742c2bd222e3c7d838c3f6e585c275688fca58